apps/web/src/pages/integrations/components/Modal/ConnectIntegrationForm.tsx

```diff
@@ -0,0 +1,379 @@
+              <a href={provider?.docReference} target="_blank" rel="noreferrer" style={{ color: '#DD2476 ' }}>
+                here.
+              </a>
+            </Text>
+          </Center>
+        </Stack>
+        <ActionIcon mt={-50} variant={'transparent'} onClick={onClose} data-test-id="connection-integration-form-close">
+          <Close />
+        </ActionIcon>
+      </Header>
+
+      <CenterDiv>
+        {provider?.credentials.map((credential: IConfigCredentials) => (
+          <InputWrapper key={credential.key}>
+            <Controller
+              name={credential.key}
+              control={control}
+              render={({ field }) => (
+                <IntegrationInput credential={credential} errors={errors} field={field} register={register} />
+              )}
+            />
+          </InputWrapper>
+        ))}
+        {webhookSupportStatus &&
+          provider?.channel &&
+          [ChannelTypeEnum.EMAIL, ChannelTypeEnum.SMS].includes(provider?.channel) && (
+            <InputWrapper>
+              <Input
+                label="Webhook URL"
+                value={webhookUrl}
+                readOnly
+                rightSection={
+                  <CopyWrapper onClick={() => webhookUrlClipboard.copy(webhookUrl)}>
+                    {webhookUrlClipboard.copied ? <Check /> : <Copy />}
+                  </CopyWrapper>
+                }
+              />
+            </InputWrapper>
+          )}
+        <Stack my={20}>
+          <ActiveWrapper active={isActive}>
+            <Controller
+              control={control}
+              name="isActive"
+              render={({ field }) => (
+                <Switch checked={isActive} data-test-id="is_active_id" {...field} onChange={handlerSwitchChange} />
+              )}
+            />
+            <StyledText data-test-id="connect-integration-form-active-text">
+              {isActive ? 'Active' : 'Disabled'}
+            </StyledText>
+          </ActiveWrapper>
+          {provider?.channel === ChannelTypeEnum.EMAIL && checkIntegrationState.isShowSwitch && (
+            <CheckIntegrationWrapper>
+              <Controller
+                control={control}
+                name="check"
+                render={({ field }) => (
+                  <Switch
+                    checked={checkIntegrationState.check}
+                    data-test-id="is_check_integration_id"
+                    {...field}
+                    onChange={handlerCheckIntegrationChange}
+                  />
+                )}
+              />
+              <StyledText>Verify provider credentials</StyledText>
+            </CheckIntegrationWrapper>
+          )}
+
+          {checkIntegrationState.isShowAlert && (
+            <Alert icon={<WarningOutlined size={16} />} title="An error occurred!" color="red" mb={30}>
+              {checkIntegrationState.errorMsg}
+            </Alert>
+          )}
```

Comment:
The error is not cleared when I do switch between different providers.
Also, it's not visible because of the scroll, I think that we should scroll to it or show the error in the banner.